<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowConsultaLocacao</class>
 <widget class="QMainWindow" name="MainWindowConsultaLocacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta Locação</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>86</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSituacaoLocacao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>169</y>
      <width>57</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Locado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeClienteLocacao">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>58</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>30</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Placa do veiculo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>142</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilometragem:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKmInicial">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>142</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>114</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor da locação:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValorLocacao">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>114</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataLocacao">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>86</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>58</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do cliente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlacaLocacao">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>30</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
